src/TimeSteppers/runge_kutta_3.jl

```diff
@@ -104,6 +105,8 @@ function time_step!(model::AbstractModel{<:RungeKutta3TimeStepper}, Δt; callbac
     calculate_pressure_correction!(model, first_stage_Δt)
     pressure_correct_velocities!(model, first_stage_Δt)
 
+    correct_advection!(model, Δt)
```

Comment:
what's wrong about it

Reply:
that it is only upwind first order, definitely incorrect 😄

Reply:
maybe "refine advective tendencies" is the right word?